src/server/multi_command_squasher.cc

```diff
@@ -15,6 +16,8 @@
 #include "server/transaction.h"
 #include "server/tx_base.h"
 
+ABSL_FLAG(size_t, throttle_squashed, 0, "");
```

Comment:
maybe squashed_reply_size_limit
add flag description
also I think we should have a default limit here maybe 128_MB ?